pipelines/push-disk-images-to-marketplaces/push-disk-images-to-marketplaces.yaml

```diff
@@ -0,0 +1,260 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: push-disk-images-to-marketplaces
+  labels:
+    app.kubernetes.io/version: "1.2.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.1.0"
```

Comment:
0.12.1

Reply:
Thanks, fixed!

Reply:
It was changed to 1.2.0, should be 0.12.1

Reply:
Oops, sorry! Fixed now